docs/guides/universal-receiver-delegate/create-custom-urd-1.md

```diff
@@ -0,0 +1,303 @@
+---
+sidebar_label: 'Create a custom Forwarder URD (1/2)'
+sidebar_position: 1
+---
+
+# Create a custom forwarder URD
+
+In this guide, we will create a custom [Universal Receiver Delegate](../../standards/generic-standards/lsp1-universal-receiver-delegate.md) contract. This contract will be called each time the associated UP receives a [LSP7 token](../../standards/nft-2.0/LSP7-Digital-Asset.md), and will forward a certain percentage to another address. The use-case it answers is:
+
+> **"As a UP owner, I want to transfer part of the token I received to another UP"**.
+
+An example scenario could be: "each time I receive USDT, I want to automatically transfer 20% to my wife's UP".
+
+## Requirements
+
+In order to follow this guide, you'll need to:
+
+1. Install the [UP Browser extension](../browser-extension/install-browser-extension.md).
+2. Fund the EOA that controls your UP (You can find this address in the extension if you click on the controller tab > "UP Extension") using the [Testnet Faucet](https://faucet.testnet.lukso.network/).
+3. Follow the contracts [Getting started](../../contracts/getting-started.md) to setup a new Hardhat project.
+
+## 1 - EOA permission
+
+First, we will need to give the EOA that controls our UP proper permission to add / edit an Universal Receiver Delegate (called "Automation").
+
+To do that, click on the controller tab > "UP Extension" which will bring the controller information page.
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/guides/lsp1/ControllerSettings.png"
+    alt="Screenshot of the controller settings in the extension"
+/>
+</div>
+
+Scroll down to the "Administration & Ownership" part and check both "Add notifications & automation" and "Edit notifications & automation".
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/guides/lsp1/ControllerPerm.png"
+    alt="Screenshot of the controller permissions in the extension"
+/>
+</div>
+
+Confirm the changes and submit the transaction.
+
+## 2 - Environment variables
+
+In your hardhat project, create a `.env` file (if it's not already present) and fill the `PRIVATE_KEY` and `UP_ADDR` with the info coming from your UP Browser Extension. To get those values:
+
+- Click on the extension
+- Click on the ⚙️ at the top right corner, then select "reveal private keys"
```

Comment:
In the new version of the browser extension, the "reveal private key" is not there anymore.

<img width="406" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/151d6b6c-ba7e-4e78-b262-0f28be175e42">

<img width="399" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/97475790-67b1-4ee5-808f-5bb048df131e">

I guess it might be somewhere else?

Reply:
the button has been removed because it needed some more work. I guess I should state that this guide has been made with v1.7.0